client/src/app/metadata-schema-form/form-item/form-item.component.html

```diff
@@ -1,7 +1,7 @@
 <div [ngClass]="rootClass">
   <div [ngClass]="{'label': true, 'read-only': data.disabled}">
-    <label class="vf-form__label">{{data.label}}</label>
-    <app-vf-asterisk *ngIf="!data.disabled" [isRequired]="data.isRequired"></app-vf-asterisk>
+    <label class="vf-form__label" *ngIf="data.disabled || !rootClass['vf-form__item--checkbox']">{{data.label}}</label>
```

Comment:
I would make it DRYer, extract a function and use it here and in the next line.

Reply:
A function that would only be used once?
The next line is a different condition, should I make a function for that too?